src/main/webapp/js/studentProfile.js

```diff
@@ -62,9 +62,7 @@ function finaliseUploadPictureForm(event) {
         	$('#statusMessage').css("display", "block")
         	        .attr('class', 'alert alert-danger')
         	        .html('There seems to be a network error, please try again later');
-        	$("html, body").animate({
-        		scrollTop: 0
-        	});
+        	scrollToTop({duration: ''});
```

Comment:
Indeed, maybe we could either put in strings with some meaning like 'fast', 'slow', 'normal' instead of passing in empty strings because empty strings are falsy and doesn't have much meaning or we could put in comments, do which you think is better suited for this PR?